test/quantization/test_quantize_fx.py

```diff
@@ -2361,6 +2362,40 @@ def test_rnn(self):
                                      [100, -155]], dtype=torch.float).unsqueeze(0).repeat(niter, 1, 1)
         self._test_rnn_impl(qconfigs, RNNDynamicModel, module_type_strs, module_types, sample_input)
 
+    def _test_conv_transpose_impl(
```

Comment:
can we use checkGraphModeFxOp? https://github.com/pytorch/pytorch/blob/master/test/quantization/test_quantize_fx.py#L1368
this is the common check function we use for all modules, feel free to add more functionalities there so other tests can benefit as well.